package/MDAnalysis/visualization/streamlines_3D.py

```diff
@@ -245,10 +245,11 @@ def point_in_cube(array_point_coordinates, list_cube_vertices, cube_centroid):
         cube_half_side_length = scipy.spatial.distance.pdist(array_cube_vertices, 'euclidean').min() / 2.0
         array_cube_vertex_distances_from_centroid = scipy.spatial.distance.cdist(array_cube_vertices,
                                                                                  cube_centroid[np.newaxis, :])
-        np.testing.assert_almost_equal(array_cube_vertex_distances_from_centroid.min(),
-                                          array_cube_vertex_distances_from_centroid.max(), decimal=4,
+        np.testing.assert_allclose(array_cube_vertex_distances_from_centroid.min(),
+                                          array_cube_vertex_distances_from_centroid.max(), rtol=0, atol=1.5*(1e-4),
```

Comment:
Write floats as `1.5e-4` and not as `1.5*(1e-4)`.

This applies to all floats with powers of ten throughout.